staging/src/k8s.io/apiserver/pkg/admission/metrics/metrics.go

```diff
@@ -196,6 +197,16 @@ func newAdmissionMetrics() *AdmissionMetrics {
 		},
 		[]string{"name", "type", "operation", "error_type", "rejection_code"})
 
+	webhookFailOpen := metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Namespace:      namespace,
+			Subsystem:      subsystem,
+			Name:           "webhook_fail_open_count",
```

Comment:
This must be suffixed `_total` as per instrumentation guidelines.

Reply:
@logicalhan let me fix that, will open a PR shortly adding you are reviewer

Reply:
@logicalhan the file contains 2 metrics:
webhook_rejection_count (preexisting)
webhook_fail_open_count (new added by me)
Do you want me to address both? renaming them to:
webhook_rejection_total
webhook_fail_open_total